src/main/java/com/io/image/manager/service/ImageServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.io.image.manager.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.net.URL;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ImageServiceImpl implements ImageService {
+    @Value("${image-manager.origin-server}")
+    private String originServer;
```

Comment:
I think in future tasks we will have more values ​​in yml file so I don't know if it would be better to create a new class for that, what you think about it?

Reply:
Do you mean a config class and then inject it in this service to get access to the variable?

Reply:
Exactly